src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -2482,11 +2445,10 @@ void EmitSingleChar(RegexNode node, bool emitLengthCheck = true, string? offset
 
                 if (node.IsSetFamily)
                 {
-                    expr = $"{MatchCharacterClass(hasTextInfo, options, expr, node.Str!, IsCaseInsensitive(node), negate: true, additionalDeclarations, requiredHelpers)}";
+                    expr = $"{MatchCharacterClass(options, expr, node.Str!, negate: true, additionalDeclarations, requiredHelpers)}";
```

Comment:
As long as you're changing this, you can remove the superfluous use of string interpolation:
```suggestion
                    expr = MatchCharacterClass(options, expr, node.Str!, negate: true, additionalDeclarations, requiredHelpers);
```